bgpug/templates/base.html

```diff
@@ -8,6 +8,22 @@
         <meta charset="utf-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="robots" content="all" />
+        <meta name="city" content="Sofia"/>
+        <meta name="country" content="Bulgaria"/>
+        <meta name="resource-type" content="document"/>
+        <meta name="distribution" content="global"/>
+        <meta name="allow-search" content="yes"/>
+        <meta name="rating" content="general"/>
+        <meta name="audience" content="General"/>
+        <meta name="Description" content="PosgreSQL България"/>
+
+        <meta property="og:locale" content="bg_BG" />
+        <meta property="og:type" content="website" />
+        {% block extra_meta %}
+        <meta property="og:url" content="http://bgpug.org"/>
+        {% endblock %}
```

Comment:
It's OK, I'll just check zinnia's original templates for meta properties blocks/templatetags that we can use instead.